telluric/vrt.py

```diff
@@ -136,4 +152,87 @@ def boundless_vrt_doc(
         dst_window = Window(xoff, yoff, xsize, ysize)
         vrt.add_band_simplesource(mask_band, 'mask,1', 'Byte', False, src_dataset.name,
                                   width, height, block_shape[1], block_shape[0], src_window, dst_window)
-    return vrt.tostring()
+    return vrt
+
+
+def band_name_to_color_interpretation(band_name):
+    if band_name.lower() in ['red', 'green', 'blue']:
+        return band_name
+    if band_name.lower().endswith('_enhanced'):
```

Comment:
adding satellogic logic here, is not good 

bands that have `_enhanced` suffixes exist only in satellogic

Reply:
yes